### ISSUE TITLE
Can no longer nest custom models

### ISSUE THREAD

**Issue by zefie** [open-webui/open-webui#3196]
# Bug Report

## Description

**Bug Summary:**
Before the recent update that split the custom model form into separate inputs, I was able to "nest" models. E.g. create a custom model "A" `FROM: mistral`, then create a custom model "B" `FROM: A` to extend the model. I do not see custom-made models in the dropdown on the custom model creation page, thus no longer allowing nested models. The update also causes all of my nested models to show as a blank base model (since it can't find the correct model in the menu).

**Steps to Reproduce:**
Create model named "A"
Create model named "B"
Try to base B from A.

**Expected Behavior:**
All available models to appear in the model dropdown

**Actual Behavior:**
Only actual base models appear

## Environment

- **Open WebUI Version:** 0.3.4
- **Ollama (if applicable):** 0.1.44

- **Operating System:** Windows 11 w/ WSL2 Ubuntu 22.04
- **Browser (if applicable):** Google Chrome (irrelevant)

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
N/A

**Docker Container Logs:**
N/A

**Screenshots (if applicable):**
N/A

## Installation Method

Docker compose with ollama and open-webui in separate containers.

**Comment by Peter-De-Ath** [comment]
I spotted this too, as I'd also done the same thing, so understand how now the base being blank for previous Custom models is frustrating.

These are not actual models being created in Ollama though, so they aren't actually extending or using FROM:
These custom models are internal to Open WebUI.

When creating B from A the base model will still technically just be the base model of A.

I think how it is now, not showing the custom models in the base models list is better, as you don't gain anything from creating custom from custom.

You can still clone a custom model.

**Comment by zefie** [comment]
> These are not actual models being created in Ollama though, so they aren't actually extending or using FROM: These custom models are internal to Open WebUI.

I think that is the thing. They used to be. On whatever old 0.2.x version I had, before the form change, it would create a model on the ollama side, as I was able to access my custom models using my own third-party interface (directly with ollama API). But now they do not show up in that tool, because they are internal to Open WebUI. 

An option to toggle the old behavior would be appreciated, as I have found it useful to create Ollama profiles directly.

Or I guess I could use the old version to create custom models and just use them on the new version.
